packages/cells/src/widget.ts

```diff
@@ -690,6 +690,13 @@ class MarkdownCell extends Cell {
   readonly model: IMarkdownCellModel;
 
   /**
+   * A promise that resolves when the widget renders for the first time.
+   */
+  get ready(): Promise<void> {
```

Comment:
What if the widget is updated and gets into a state that requires re-rendering? Can that ever happen? How does the promise get reset?